galsim/chromatic.py

```diff
@@ -2223,12 +2223,13 @@ def evaluateAtWavelength(self, wave):
 class ChromaticFourierSqrtProfile(ChromaticObject):
     """A class for computing the Fourier-space square root of a ChromaticObject.
 
-    The ChromaticFourierSqrt class represents a wavelength-dependent Fourier-space square root of a profile.
+    The ChromaticFourierSqrt class represents a wavelength-dependent Fourier-space square root of
```

Comment:
FourierSqrt is the function that people would typically call that in turn constructs either a FourierSqrtProfile or ChromaticFourierSqrtProfile class.  So the user should typically only need to write FourierSqrt for any use case.

This is analogous to Convolve, Convolution and ChromaticConvolution.  or Add, Sum, ChromaticSum.  Just in this case, FourierSqrt is less obviously a verb.

But certainly your first point is correct.  There is no such thing as ChromaticFourierSqrt.